static/app/data/platformCategories.tsx

```diff
@@ -121,6 +122,9 @@ export const desktop = [
   'native-qt',
   'minidump',
   'unity',
+  'flutter',
+  'dart-flutter',
```

Comment:
I believe it's just `flutter` as it's a top level platform: https://github.com/getsentry/sentry-docs/tree/master/src/wizard/flutter

```suggestion
```

Reply:
the idea is having pure `dart` as also desktop, similar to only `java`, also available for Desktop.

Reply:
@bruno-garcia are you fine having dart as desktop too?

Reply:
I'm not sure it's worth adding Dart as desktop. Java is there because we don't have JavaFX, Swing or something else.
In the case of Dart, it was mainly a web tech (even though it'd kind of dead now after they announced no more development for it standalone).

I'd say Flutter on Desktop makes total sense, but I'd leave out Dart which has no Desktop UI framework (that I'm aware of)

Reply:
its possible but not as easy/standard as flutter, makes sense, removing it.